packages/app/ui/toast/Toast.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import { useEffect, ReactNode } from "react";
+import { cva, cx } from "class-variance-authority";
+
+import { Button, Icon, Modal, ModalBaseProps, TitleText } from "@/ui";
+
+export enum ToastPosition {
+  BOTTOM_LEFT = "bottom-left",
+  BOTTOM_RIGHT = "bottom-right",
+  TOP_LEFT = "top-left",
+  TOP_RIGHT = "top-right",
+}
+
+export enum Severity {
+  ERROR = "warning",
+  SUCCESS = "check",
+}
+
+interface ToastProps extends ModalBaseProps {
+  autoClose?: boolean;
+  autoCloseTime?: number;
+  children: ReactNode;
+  position?: ToastPosition;
+  severity?: Severity;
+  title: string;
+}
+
+export const Toast = ({
+  autoClose,
+  autoCloseTime = 3000,
+  children,
+  closeAction,
+  isOpen,
+  position,
+  severity,
+  title,
+}: ToastProps) => {
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (autoClose && autoCloseTime && isOpen) {
+        closeAction();
+      }
+    }, autoCloseTime);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [autoClose, autoCloseTime, closeAction, isOpen]);
+
+  const hasSeverity = Boolean(severity);
+
+  return (
+    <Modal
+      className={cx("py-6 pl-6 pr-4 !w-fit", toastStyles({ position }))}
```

Comment:
`"py-6 pl-6 pr-4 !w-fit"` belong to default toastStyles, right?

Reply:
Yes! Exactly :D

OIC maybe I could move them into `toastStyles`